### DISCUSSION TITLE
无法启动UnblockNeteaseMusic服务端

### DISCUSSION THREAD

**Discussion by 9iulii** [UnblockNeteaseMusic/install-scripts#14] [Q&A]
求助，设置完配置文件之后服务端启动失败，环境为centos7.5

**Comment by 1715173329** [comment]
给下日志

**Reply by 9iulii** [reply]
需要提供什么日志呢，我这边只有错误代码

**Reply by pan93412** [reply]
不然你錄影一下你的完整操作過程？

**Reply by 1715173329** [reply]
有什么贴什么

**Reply by 9iulii** [reply]
好的，稍等

**Reply by 1715173329** [reply]
看起来是 CentOS7 的 systemd 不支持 is-active 命令。
lsof -i:7999 看看

**Reply by 9iulii** [reply]
COMMAND  PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
node    1715 root   18u  IPv4  27566      0t0  TCP *:irdmi2 (LISTEN)

**Reply by 1715173329** [reply]
服务端已经起来了，显示问题。
我抽空换个判断方法。

**Reply by 9iulii** [reply]
好的，麻烦您了，谢谢。
